src/platform/tests/Bridge/OpenAi/TokenOutputProcessorTest.php

```diff
@@ -70,8 +72,11 @@ public function testItAddsRemainingTokensToMetadata()
         $processor->processOutput($output);
 
         $metadata = $output->result->getMetadata();
+        $tokenUsage = $metadata->get('token_usage');
```

Comment:
What about having a dedicated setTokenUsage/getTokenUsage on the metadata object?